packages/components/src/components/combobox/shadow.tsx

```diff
@@ -192,12 +195,17 @@ export class KolCombobox implements ComboboxAPI {
 									spellcheck="false"
 									{...this.controller.onFacade}
 									onInput={this.onInput.bind(this)}
-									onClick={this.toggleListbox.bind(this)}
 									onChange={this.onChange.bind(this)}
 									placeholder={this.state._placeholder}
 								/>
-								<span class={{ combobox__icon: true }}>
-									<KolIconTag _icons="codicon codicon-triangle-down" _label={translate('kol-dropdown')} onClick={this.toggleListbox.bind(this)} />
+								<span
+									role="button"
```

Comment:
Ist die Änderung seitens Barrierefreiheit besprochen bzw. getestet? Ich dachte, es sollte explizit ein span ohne Rolle sein, sonst könnten wir ja auch eine `<button>` nehmen?

Reply:
> Das heißt die Info war nur einen span ohne Attribute zu verwenden?

Ich habe da keine weiteren Infos zu, glaube aber dass sich Cyrine bei der Initialentwicklung an einer funktionierenden Komponente orientiert hat und das von unserem Barrierefreiheitsteam mittlerweile so abgenommen wurde. Würde ohne Not also lieber keine Änderungen an am A11y-Tree machen.